cmd/snap/cmd_interfaces.go

```diff
@@ -86,6 +87,10 @@ func (x *cmdInterfaces) Execute(args []string) error {
 	if len(ifaces.Plugs) == 0 && len(ifaces.Slots) == 0 {
 		return fmt.Errorf(i18n.G("no interfaces found"))
 	}
+
+	defer fmt.Fprintf(Stderr, "\n%s\n",
+		i18n.G("The 'snap interfaces' command is deprecated. Use 'snap connections' instead."))
```

Comment:
To me adding a "please" feels more friendly but maybe it's overdoing it (WDYT @degville ?). So "The 'snap interfaces' command is deprecated. Please use 'snap connections' instead." sounds a tiny bit better to me.

Reply:
> To me adding a "please" feels more friendly but maybe it's overdoing it

This is a tough one. And I did think through the same thing. But to me - and it's super subtle -  there's an element in *please* that is the opposite. It's tiny bit prescriptive, condescending and commanding - such as an authority asking for your papers ("Please can I see your papers..."). Without please, it's not impolite or unfriendly, it's just the shortest way of providing the most helpful information.

Reply:
Would changing the `Use` to a `Try` accomplish the friendlification without loading the prescriptive?

Reply:
I think `Try`makes us look uncertain and lacking in confidence. It could imply *we removed a feature, and we hope the connections command is up to the task of replacing it, but we're not sure*.

Honestly, my feelings are not strong about this - adding *please* is absolutely fine and I'm just throwing my opinion in. I can totally see why something friendlier is preferable. And they're only words :)

Maybe `Take a look at...`?
Or `The 'snap interfaces' command is deprecated and has been replaced with the 'snap connections' command.`

Reply:
That last suggestion sounds most neutral to me. I think we could go with that.

Reply:
+1 to that last one!